src/plugins/AxiosPlugin.ts

```diff
@@ -45,11 +46,14 @@ class AxiosPlugin implements SwPlugin {
       else
         config = url ? {...url} : {};
 
+      const carrier = ContextCarrier.from(config.headers || {});
       const {origin, host, pathname: operation} = new URL(config.url, config.baseURL);  // TODO: this may throw invalid URL
       const method = (config.method || 'GET').toUpperCase();
       const span = ignoreHttpMethodCheck(method)
         ? DummySpan.create()
-        : ContextManager.current.newExitSpan(operation, Component.AXIOS, Component.HTTP);
+        : carrier
+          ? ContextManager.current.newEntrySpan(operation, carrier)
```

Comment:
Why would a client be creating a server `EntrySpan`?

Reply:
The AxiosPlugin also runs in the Node layer. So I think there's a distinction to be made here, that I'm not being captured in the right way, but the problem is there.